wgpu-hal/src/dx12/device.rs

```diff
@@ -577,7 +577,9 @@ impl crate::Device for super::Device {
                 None
             },
             handle_uav: if desc.usage.intersects(
-                crate::TextureUses::STORAGE_READ_ONLY | crate::TextureUses::STORAGE_READ_WRITE,
+                crate::TextureUses::STORAGE_READ_ONLY
+                    | crate::TextureUses::STORAGE_WRITE_ONLY
+                    | crate::TextureUses::STORAGE_READ_WRITE,
```

Comment:
thought: If `STORAGE_READ_WRITE` really _is_ the union of `*_{WRITE,READ}_ONLY` like in the adapter module, then adding those bits here seems redundant, but not in a bad way.

Reply:
In this case it's not the union of the other 2. Take for example the `bgra8unorm` format: it only supports write-only storage access.

wgpu-core will set the hal usage to `TextureUses::STORAGE_WRITE_ONLY`.